### ISSUE TITLE
[Bug]: Subscriptions - subscribed channels currently does not have any videos

### ISSUE THREAD

**Issue by Tealc59** [MarmadileManteater/FreeTubeCordova#415]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Open freetube android
2. Going to Subscriptions page
3. No videos shown at all  (message "your subscribed channels currently does not have any videos")
4. Same behavior on "videos" + "shorts" + "live"
5. Going to each channel manually still show latest videos from this channel

### Expected Behavior

All latest videos from subscribed channels should appear  (with greyed videos if already watched)
Problem appeared 2-3 days ago.

### Issue Labels

accessibility issue, API issue, content not loading, feature stopped working

### FreeTube Version

v0.21.3.122 Beta

### Operating System Version

Android 14 (Samsung galaxy s24+ sm-s926b)

### Installation Method

.apk

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.21.3.122 Beta

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Tealc59** [comment]
I respond to myself,  i could find the information:

> YouTube has a limit on how many requests per day an IP address is allowed to make. If you exceed this limit, you will be greeted with a 429 error and will not be allowed to browse using FreeTube any more. This limit can be reached much faster if you have a large amount of subscriptions

my workaround :  use "local api", feed with rss checked, wait a little
